README.md

```diff
@@ -1,2 +1,8 @@
-# cfc_pain_sounds
-OOF
+# [Deprecated] CFC Pain Sounds
+OOF! 
```

Comment:
OOOOF?! What are you crazy, we're a professional company, we don't put up with these kind of jokes and japes in OUR READMES

FIX THIS IMMEDIATELY